doc/source/whatsnew/v0.23.0.txt

```diff
@@ -700,6 +700,7 @@ Performance Improvements
 - Improved performance of :func:`DataFrameGroupBy.rank` (:issue:`15779`)
 - Improved performance of variable ``.rolling()`` on ``.min()`` and ``.max()`` (:issue:`19521`)
 - Improved performance of ``GroupBy.ffill`` and ``GroupBy.bfill`` (:issue:`11296`)
```

Comment:
can you add refs to these (and the ffill/bfill ones)

Reply:
Right now I believe the only way to get a GroupBy link to render is to provide the full namespace, i.e. `pandas.core.groupby.GroupBy.any`. Is that what you are looking for or are you looking for just `GroupBy.any` to render? I'm assuming that's possible by modifying conf.py but would need to do some more digging

Reply:
oh I c, yes would like these to render (specifying the full path is ok)